### PR TITLE
Some license fixes

### PR DESCRIPTION
**Pull request by Piraty**

### CODE REVIEW

File: srcpkgs/pam/template [hunk 1]
```diff
@@ -26,7 +26,7 @@ lib32disabled=yes
 short_desc="A flexible mechanism for authenticating users"
 maintainer="Juan RP <xtraeme@voidlinux.org>"
 homepage="http://www.kernel.org/pub/linux/libs/pam/"
```
Comment (maxice8) [inline srcpkgs/pam/template]: move to under license=

File: srcpkgs/pam/template [hunk 2]
```diff
@@ -26,7 +26,7 @@ lib32disabled=yes
 short_desc="A flexible mechanism for authenticating users"
```
Comment (maxice8) [inline srcpkgs/pam/template]: remove article

File: srcpkgs/open-vm-tools/template [hunk 3]
```diff
@@ -14,7 +14,7 @@ makedepends="fuse-devel glib-devel icu-devel libdnet-devel libmspack-devel
  $(vopt_if x11 'gtkmm-devel libXtst-devel xmlsec1-devel')"
 short_desc="The open source implementation of VMware Tools"
```
Comment (maxice8) [inline srcpkgs/open-vm-tools/template]: remove article

File: srcpkgs/parted/template [hunk 4]
```diff
@@ -1,15 +1,15 @@
...
 pkgname=parted
 version=3.2
-revision=6
+revision=7
 build_style=gnu-configure
 # parted wants off_t as 64bit type
 configure_args="ac_cv_sizeof_off_t=8"
 hostmakedepends="pkg-config"
 makedepends="libuuid-devel libblkid-devel ncurses-devel readline-devel device-mapper-devel"
 short_desc="The GNU Parted disk partition resizing program"
```
Comment (maxice8) [inline srcpkgs/parted/template]: remove article

File: srcpkgs/parted/template [hunk 5]
```diff
@@ -1,15 +1,15 @@
...
 homepage="http://www.gnu.org/software/parted/"
```
Comment (maxice8) [inline srcpkgs/parted/template]: move under license=

File: srcpkgs/parted/template [hunk 6]
```diff
@@ -1,15 +1,15 @@
...
-license="GPL-3"
+license="GPL-3.0-or-later"
 maintainer="Juan RP <xtraeme@voidlinux.org>"
```
Comment (maxice8) [inline srcpkgs/parted/template]: move above license=

**Comment by maxice8** [comment]
please notify when proposed changes are made

**Comment by Piraty** [comment]
> please notify when proposed changes are made

Will do, thank you.

**Pull request merged**